### DISCUSSION TITLE
反查

### DISCUSSION THREAD

**Discussion by qwert-f** [fcitx5-android/fcitx5-android#440] [Q&A | 问答]
看rime插件目录的build里的schema.yaml里配置了reverse_lookup当前要怎么反查呀，punctuator里配置的符号这些怎么快速使用

**Comment by rocka** [comment]
请自行查找文档学习如何配置和使用 rime 。如果你从未在其它平台上使用过 rime ，不建议把 fcitx5-android rime plugin 作为学习 rime 的方式。

**Reply by qwert-f** [reply]
是似乎其中配置开𡰪反查的按键符号并没有办法响应和我用没用过其它rime平台无关。在开头已经说明是rime plugin的build目录中的文件甚至不关我任何事。如果plugin对反查功能并不支持或支持还不完善，应当删除build中方案文件相应配置或其注释避免不必要的误解

**Reply by MokOopsing** [reply]
自行修改反查引导符号，而不应该删预置方案，你不会用，别人会用

**Comment by eagleoflqj** [comment]
rime默认的反查键是反引号`，位于物理键盘按键1的左边。
在fcitx5-android的当前版本的虚拟键盘中，长按z，弹出的6个符号里左下角那个就是，即可在拼音和笔画之间反查。

**Reply by qwert-f** [reply]
感谢，我说咋个我配了用那个符号开启反查咋出不来，原来是那个符号需要这样才算输入了
